scene/main/window.cpp

```diff
@@ -881,10 +881,6 @@ bool Window::_can_consume_input_events() const {
 }
 
 void Window::_window_input(const Ref<InputEvent> &p_ev) {
-	if (Engine::get_singleton()->is_editor_hint() && (Object::cast_to<InputEventJoypadButton>(p_ev.ptr()) || Object::cast_to<InputEventJoypadMotion>(*p_ev))) {
-		return; //avoid joy input on editor
-	}
```

Comment:
There's potential for some issues with this, as gamepad input is sent to all applications by the OS, so when you're testing your game with a gamepad, you might also end up changing stuff by mistake in the editor since default `ui_*` actions handle gamepad too.

But I guess let's merge and test to see what happens?

Reply:
Yeah, I guess at the end of the day the current "select a button/axis from a menu" is there anyway, so reverting this shouldn't be an issue. It is very nice to have but if it causes too many issues in the editor then it should be reverted.

Nothing else should even be bound to joypad stuff in the editor though.